lib/datadog/tracing/contrib/active_support/cache/events/cache.rb

```diff
@@ -81,7 +81,7 @@ def on_start(span, event, _id, payload)
 
                 span.set_tag('EVENT', event)
 
-                set_cache_key(span, key, mapping[:multi_key])
+                set_cache_key(span, key, mapping[:multi_key]) if Datadog.configuration.tracing[:active_support][:cache_key_enabled]
```

Comment:
There's a shorthand to access the integration configuration that you can use here:
```suggestion
                set_cache_key(span, key, mapping[:multi_key]) if configuration[:cache_key_enabled]
```

It is originally defined here: https://github.com/DataDog/dd-trace-rb/blob/ee50bd2a817b6e612692f86d9d7e85acc3a89f9c/lib/datadog/tracing/contrib/active_support/cache/event.rb#L23-L25